Sources/Apollo/GraphQLGETTransformer.swift

```diff
@@ -44,7 +44,7 @@ struct GraphQLGETTransformer {
     queryItems.append(URLQueryItem(name: self.queryKey, value: query))
     components.queryItems = queryItems
     
-    guard let variables = self.body.jsonObject[self.variablesKey] as? [String: AnyHashable] else {
+    guard let variables = self.body.jsonObject[self.variablesKey] as? [AnyHashable : Any] else {
```

Comment:
So messing around in a playground then looking at [the docs for `JSONSerialization`](https://developer.apple.com/documentation/foundation/jsonserialization) I think this needs to be `[String: Any]` - JSON Serialization barfs if the keys aren't strings.